AbletonMCP_Remote_Script/__init__.py

```
@@ -561,6 +736,314 @@ def _set_tempo(self, tempo):
+                "new_value": target_param.value
+            }
+            return result
+        except Exception as e:
+            self.log_message("Error setting device parameter: " + str(e))
+            raise
+    
+    def _load_effect_to_chain(self, track_index, rack_device_index, chain_index, effect_uri):
+        """Load an effect into a specific chain of a rack using move_device"""
+        try:
+            if track_index < 0 or track_index >= len(self._song.tracks):
+                raise IndexError("Track index out of range")
```

Ruff (0.14.6)

Abstract `raise` to an inner function

(TRY301)